src/components/Pool/PairView/UserLiquidity/UserLiquidity.container.tsx

```diff
@@ -0,0 +1,104 @@
+      : undefined
+
+  const [token0Deposited, token1Deposited] = !!pair
+    ? !!totalPoolTokens &&
+      totalPoolTokens.greaterThan('0') &&
+      !!userPoolBalance &&
+      userPoolBalance.greaterThan('0') &&
+      // this condition is a short-circuit in the case where useTokenBalance updates sooner than useTotalSupply
+      JSBI.greaterThanOrEqual(totalPoolTokens.raw, userPoolBalance.raw)
+      ? [
+          pair.getLiquidityValue(pair.token0, totalPoolTokens, userPoolBalance, false),
+          pair.getLiquidityValue(pair.token1, totalPoolTokens, userPoolBalance, false),
+        ]
+      : [new TokenAmount(pair.token0, '0'), new TokenAmount(pair.token1, '0')]
+    : [undefined, undefined]
+
+  return (
+    <DimBlurBgBox padding={'24px'}>
+      <Flex flexDirection={['column', 'row']} alignItems="center" justifyContent="space-between" paddingBottom={'24px'}>
+        <Text fontSize="16px" mb={['12px', '24px']}>
+          {t('yourLiquidity')}
+        </Text>
+        <Box>
+          <ButtonExternalLink link={getAccountAnalyticsLink(account || '', chainId)}>
+            {t('accountAnalytics')}
+          </ButtonExternalLink>
+        </Box>
+      </Flex>
+      <Box marginBottom={3}>
+        <InfoGrid>
+          <ValueWithLabel
+            title={t('poolShare')}
+            value={poolTokenPercentage ? poolTokenPercentage.toFixed(2) + '%' : '-'}
+          />
+          <ValueWithLabel title={t('poolTokens')} value={userPoolBalance ? userPoolBalance.toSignificant(4) : '-'} />
+          <ValueWithLabel
+            title={t('pooledToken', { token: currency0?.symbol })}
+            value={token0Deposited ? token0Deposited.toSignificant(6) : '-'}
+          />
+          <ValueWithLabel
+            title={t('pooledToken', { token: currency1?.symbol })}
+            value={token1Deposited ? token1Deposited.toSignificant(6) : '-'}
+          />
+          <ValueWithLabel
+            title={t('swapFee')}
+            value={
+              pair ? new Percent(JSBI.BigInt(pair.swapFee.toString()), JSBI.BigInt(10000)).toSignificant(3) + '%' : '-'
+            }
+          />
+        </InfoGrid>
+      </Box>
+      <Flex flexDirection={['column', 'row']} alignItems="center">
+        <Box marginBottom={[3, 0]} width="100%">
+          <ButtonPurpleDim
+            as={Link}
+            to={currency0 && currency1 ? `/pools/add/${currencyId(currency0)}/${currencyId(currency1)}` : ''}
+          >
+            {t('addLiquidity')}
+          </ButtonPurpleDim>
+        </Box>
+        <Box marginLeft={[0, 3]} width="100%">
+          <ButtonPurpleDim
+            disabled={token0Deposited?.equalTo('0')}
+            as={token0Deposited?.equalTo('0') ? ButtonPurpleDim : Link}
+            to={currency0 && currency1 ? `/pools/remove/${currencyId(currency0)}/${currencyId(currency1)}` : ''}
+          >
+            {t('removeLiquidity')}
+          </ButtonPurpleDim>
+        </Box>
+      </Flex>
+    </DimBlurBgBox>
```

Comment:
I would love us to swap to [tailwind css](https://tailwindcss.com/), Box and flex are nice, but it's not so easy to do responsive design and styles, tailwind makes it a breeze. I think we would gain a lot, and could transition incrementally.  We could get rid of styled components and rebass entirely.

Dunno if you have an opinion on that?

Reply:
Yes! Tailwind seems amazing I was actually looking into it couple of weeks ago I think zett was looking at it also. Its a big learning curve and switch but we should def consider and look more into it. Only thing I don't like is how it has those long class names with which you style...

Reply:
I also agree for a separate UI folder for components. What about moving to https://mui.com/ or https://flowbite.com/ instead of building a new design system.

Reply:
I'm a big fan of MUI and and for sure it would help to clean up codebase, however it also has some learning curve.

Reply:
Never worked with MUI but the problem with full set libs is that to do custom styles sometimes it's much harder, you have to fight the lib, something we don't want. 

@wixzi I liked [flowbit.com](https://flowbit.com/) (it's kind of like [tailwindui.com/](https://tailwindui.com/)) because they are built upon tailwind which makes it easy to do custom with a design system that backs you up.

@Mi-Lan is not so big learning curve, you get good pretty quickly, they have extensions do Vscode that help a lot with auto complete and the docs are pretty good. The good about long class names is that you can see how the component looks and as we work with components it'll be contained in each component. So when we use components we don't have to care about that.